From 5583f651d07532729bdaf9d65ede9bea18e38b5d Mon Sep 17 00:00:00 2001
From: Laurent Vivier <lvivier@redhat.com>
Date: Mon, 28 Sep 2020 11:06:48 -0400
Subject: [PATCH 5/6] migration: increase max-bandwidth to 128 MiB/s (1 Gib/s)

RH-Author: Laurent Vivier <lvivier@redhat.com>
Message-id: <20200928110648.342290-1-lvivier@redhat.com>
Patchwork-id: 98494
O-Subject: [RHEL-AV-8.3.0 qemu-kvm PATCH] migration: increase max-bandwidth to 128 MiB/s (1 Gib/s)
Bugzilla: 1874004
RH-Acked-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
RH-Acked-by: Daniel P. Berrange <berrange@redhat.com>
RH-Acked-by: Greg Kurz <gkurz@redhat.com>

max-bandwidth is set by default to 32 MiB/s (256 Mib/s)
since 2008 (5bb7910af031c).

Most of the CPUs can dirty memory faster than that now,
and this is clearly a problem with POWER where the page
size is 64 kiB and not 4 KiB.

Signed-off-by: Laurent Vivier <lvivier@redhat.com>
Message-Id: <20200921144957.979989-1-lvivier@redhat.com>
Reviewed-by: David Gibson <david@gibson.dropbear.id.au>
Reviewed-by: Greg Kurz <groug@kaod.org>
Reviewed-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
Signed-off-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
(cherry picked from commit 7590a2ae091fde8bb72d5df93977ab9707e23242)
Signed-off-by: Laurent Vivier <lvivier@redhat.com>

BREW: https://brewweb.engineering.redhat.com/brew/taskinfo?taskID=31576368
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1874004
Upstream: Merged
Tested: checked a migration while an installation is running can end
        before the end of installation
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 migration/migration.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/migration/migration.c b/migration/migration.c
index 7a89ce39a76..8193f33f768 100644
--- a/migration/migration.c
+++ b/migration/migration.c
@@ -57,7 +57,7 @@
 #include "qemu/queue.h"
 #include "multifd.h"
 
-#define MAX_THROTTLE  (32 << 20)      /* Migration transfer speed throttling */
+#define MAX_THROTTLE  (128 << 20)      /* Migration transfer speed throttling */
 
 /* Amount of time to allocate to each "chunk" of bandwidth-throttled
  * data. */
-- 
2.27.0

